sdk/core/azure-core-test/src/main/java/module-info.java

```diff
@@ -8,6 +8,7 @@
     requires com.fasterxml.jackson.datatype.jsr310;
     requires org.junit.jupiter.api;
     requires reactor.test;
+    requires org.junit.jupiter.params;
```

Comment:
Is this required? I didn't see any changes to junit params.

Reply:
`Argument` is jupiter params, which is used by:
List<Argument> getTestParametersFromServiceVersion();